guardian/mobile-n10n
# Increase chunked token batch size

## What does this change?
Increased the chunked token batch size to 5000 from 1000, to decrease the amount of invocations needed to send a notification. In the hope we can ease the pressure we have on concurrency.

| Before (invocation count) | After (invocation count) |
| --------------------------| -------------------------|
| 1600 (approx) | 400 (approx)

## How to test
Send notification before and after batch size change to see difference in invocation count. Should we double batch size to 2000 to check performance? It's hard to test true performance until in PROD, so we could roll this change, monitor and rollback if necessary.
